pkgs/applications/networking/instant-messengers/pidgin-plugins/purple-signald/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib, stdenv, fetchgit, fetchFromGitHub, pidgin, glib, json-glib, signald }:
+
+stdenv.mkDerivation rec {
+  pname = "purple-signald";
+  version = "0.11.0";
+
+  src = fetchgit {
```

Comment:
Please use `fetchFromGitHub`.

Reply:
fetchFromGithub seemed unable to fetch the submodules, when I tried before, but I can try again.